client/landing/gutenboarding/available-designs.ts

```diff
@@ -8,147 +8,14 @@ import { addQueryArgs } from '@wordpress/url';
  */
 import { isEnabled } from '../../config';
 import type { Design } from './stores/onboard/types';
+const availableDesignsConfig = require( './available-designs-config.json' );
```

Comment:
I was looking into whether we could swap this out for:

```
import availableDesignsConfig from './available-designs-config.json';
```

However it then raises a type error in `const availableDesigns: Readonly< AvailableDesigns > = `availableDesignsConfig` because it looks at the heading and base fonts as strings, instead of literals of the type `Font`. It seems that importing JSON in TypeScript doesn't allow for importing as literals, as raised in this issue: https://github.com/microsoft/TypeScript/issues/26552

As a workaround we can leave this import using `require` (which I think means that TypeScript isn't following what's happening with the `availableDesignsConfig variable)  or mess around with our types so that `AvailableDesigns` doesn't expect a FontPair with `headings` and `base` of type `Font` but instead a font pair of strings.

Since this is working as is, and we ultimately want to move this JSON to an endpoint anyway, I think I'd leave it with `require`, unless anyone has strong opinions on it.

Reply:
Thanks for investigating this @andrewserong !!!